src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -214,141 +215,143 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
       0
     )
   }
+  if (results.length == 0) {
+    return <EmptyArtistAuctionResults></EmptyArtistAuctionResults>
```

Comment:
Self closing tag: `<EmptyArtistAuctionResults />`.

I would also change the name to stay prefixed by the parent: `ArtistAuctionResultsEmptyState`